repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1683,6 +1685,12 @@ private EntityMutationResponse deleteVertices(Collection<AtlasVertex> deletionCa
         MetricRecorder metric = RequestContext.get().startMetricRecord("filterCategoryVertices");
         for (AtlasVertex vertex : deletionCandidates) {
             String typeName = getTypeName(vertex);
+
+            PreProcessor preProcessor = entityGraphMapper.getPreProcessor(typeName);
+            if (preProcessor != null) {
```

Comment:
add ensureNonAccessControlEntityType check as well

Reply:
no need for this